diode-server/proto/reconciler/v1/reconciler.proto

```diff
@@ -36,8 +36,38 @@ message RetrieveIngestionDataSourcesResponse {
   repeated IngestionDataSource ingestion_data_sources = 1;
 }
 
+// An ingestion log
+message IngestionLog {
+  string id = 1; // Unique identifier for the log
+  string state = 2; // State of the log
+  string data_type = 3; // Data type of the log
+  string request_id = 4; // Request ID associated with the log
+  int64 ingestion_ts = 5; // Timestamp of the ingestion
+  string value = 6; // The stored value in the JSON key
```

Comment:
Currently, we store original ingested entity and changeset ID after successful reconciliation but planning to store generated changeset payload as well (separately to this PR). I'd suggest to rename `value` (as too generic) with `entity` of existing proto `Entity` type to be more specific, and later on we can add `changeset` field once we have more data to work on.

Reply:
you mean to generate a dependency between reconciler and ingester? currently we do not have import ingester in reconciler proto

Reply:
done